packages/shared/src/models/survey.ts

```diff
@@ -0,0 +1,19 @@
+import type { User, UserId } from './user';
+
+export type Survey = {
+  userId: UserId;
+  description: string;
+  prevParticipation: string;
+  reasonForRetakingCourse: string | null;
+  expectations: string;
+  experience: string | null;
+  reasonToAccept: string;
+  plans: string;
+  absencePeriod: string;
+  averageTime: number;
+  associatedWords: string[];
+  courseInformationSource: string[];
+  marketingAccept: boolean;
+};
+
+export type CreateSurveyData = Omit<User, 'image'> & Survey;
```

Comment:
`Omit<User, 'image'>` do we want to change needed survey data whenever User changes? 
The fact that fields are duplicated, it's not the reason that change will always occur in those two places.